cmd/log-parser/agent.go

```diff
@@ -114,21 +114,35 @@ func unpackAgentLogEntry_v2(le LogEntry) (agent LogEntry, err error) {
 		agent.Source = "vmconsole"
 		return agent, nil
 	}
-
-	pid, err := strconv.Atoi(result["pid"])
-	if err != nil {
-		return LogEntry{}, fmt.Errorf("failed to convert pid")
+	// we were able to unpack agent msg, let's drop previous Data (runtime entries)
+	agent.Data = make(map[string]string)
+
+	for k, v := range result {
+		switch k {
+		case "container-id", "cid": // better to ensure consistency in the agent, same for exec-id/eid
```

Comment:
> same for exec-id/eid

Does this mean there should be a `case "exec-id", "eid":` as well or I'm missing something ?

Reply:
Well, you could, however the parser always has a fixed "**Container**" field for each log entry, and **exec-id** is logged only if exist (and under Data).
I could combine **eid** and **exec-id** to be mapped to the same key but i thought it may be too intrusive and may confuse a developer who is grepping for the specific naming, however i plan to change it on the agent side.

Reply:
Then maybe just don't mention **eid/exec-id** in this comment ?